helpers/calendar/events.py

```diff
@@ -118,3 +119,78 @@ def check_event_status(self, info, **kwargs):
         except AttributeError:
             raise GraphQLError(
                 "This Calendar ID is invalid")
+
+
+class CalendarEvents:
+    """
+    Sync all calendar events with Converge Database
+    :methods
+        sync_single_room_events
+        sync_all_events
+    """
+
+    def sync_single_room_events(self, room):
+        """
+        This method gets data from the calendar api
+        and syncs it with the local data
+
+        """
+        next_page = None
+        next_sync_token = None
+        while True:
+            event_results = get_google_calendar_events(
+                calendarId=room.calendar_id,
+                syncToken=room.next_sync_token,
+                pageToken=next_page)
+
+            next_page = event_results.get("nextPageToken")
+            room_events = event_results["items"]
+            next_sync_token = event_results.get("nextSyncToken")
+            for event in room_events:
+                existing_event = EventsModel.query.filter_by(
+                    event_id=event.get("id")
+                ).first()
+                participants = event.get('attendees')
+                number_of_attendees = 0
```

Comment:
Nice work @caveinn and @dnuwa . Can this conditional assignment `number_of_attendees = 0 if not participants else len(participants)` simplify the code here?

Reply:
I do not think there is a need to change, the current implementation is readable and it also avoids having the line too long.